runtime/POSIX/fd.h

```diff
@@ -23,7 +23,12 @@
 #include <sys/statfs.h>
 #endif
 
+#include <sys/stat.h>
+
 #ifdef __APPLE__
+#ifndef stat64
```

Comment:
Something like this might be more appropriate:
```suggestion
#ifdef _DARWIN_FEATURE_ONLY_64_BIT_INODE
```
See `man 2 stat` for details.

Reply:
Thanks.  I couldn't find this info using `man 2 stat` on my macOS version, but if this is a more reliable way to implement this check, a patch would be appreciated.